From eabb5f0950606c0a3734deb747bfe3184919c87f Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Thu, 30 Jul 2020 10:58:07 +0700
Subject: [PATCH] arm64: dts: r8a774a3-hihope-rzg2m: ecc dual: Reduce memory
 when enable ecc full dual

This patch will remove DRAM2 for ECC Dual channel setting when enable
full ECC

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
index 76f3109..f1f21e3 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
@@ -19,11 +19,6 @@
 		reg = <0x0 0x48000000 0x0 0x78000000>;
 	};
 
-	memory@600000000 {
-		device_type = "memory";
-		reg = <0x6 0x00000000 0x0 0x80000000>;
-	};
-
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
1.9.1

